source/get-started/primer.txt

```diff
@@ -1,8 +1,8 @@
 .. _scala-primer:
 
-======
-Primer
-======
+=======================
+Primer: Background APIs
```

Comment:
I initially thought 'background APIs' were for background tasks or something. what about just naming the page for the topics it addresses?

```suggestion
Reactive Streams and Observables
```

Or add 'Primer:' at the front if you want it to match the TOC title